### PR TITLE
Issue4 page markup and form

### LINKED ISSUE #4: Basic web page and form
Add basic page markup and form to process user email subscription preferences.

### PR DESCRIPTION
**Pull request by DeeZone**
Fixes #4 

- Build out page to follow DoSomething.org web page appearence
- Add basic form markup to allow user to adjust email subscription preferences.

### CODE REVIEW

File: mbc-user-subscriptions-server.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
 // mbc-user-subscriptions-server.js
 
 var express = require('express'),
-    exphbs  = require('express-handlebars')
+    exphbs  = require('express-handlebars'),
+    bodyparse  = require('body-parser'),
+    util = require('util')
```
Comment (sergii-tkachenko) [inline mbc-user-subscriptions-server.js]: It's bad style to leave semicolon to hang on new line. Please move it up to the end of the last statement.

File: mbc-user-subscriptions-server.js [hunk 2]
```diff
@@ -20,9 +23,30 @@ app.use(function(req, res, next) {
 })
 
 app.get('/', function(req, res, next) {
-  res.render('home');
+  if (req.query.targetEmail === undefined) {
```
Comment (sergii-tkachenko) [inline mbc-user-subscriptions-server.js]: It's better to just check the property itself, such as
```js
if (req.query.targetEmail) { }
```

Reference: http://stackoverflow.com/questions/2559318/how-to-check-for-an-undefined-or-null-variable-in-javascript
See first answer, note 2:
```js
if ( some_variable ) {
  // we don't get here if some_variable is null, undefined, 0, NaN, false, or ""
}
```

File: views/home.handlebars [hunk 3]
```diff
@@ -1 +1,23 @@
-<h1>User Subscriptions</h1>
\ No newline at end of file
+      <h1>User Subscriptions</h1>
```
Comment (sergii-tkachenko) [inline views/home.handlebars]: Why indentation from the first line?

File: views/thank-you.handlebars [hunk 4]
```diff
@@ -0,0 +1 @@
+<h1>Thank You</h1>
```
Comment (sergii-tkachenko) [inline views/thank-you.handlebars]: Please add a newline at the end of a document.

File: mbc-user-subscriptions-server.js (hunk 2 above)
Comment (DeeZone) [inline mbc-user-subscriptions-server.js]: How about:
```
if (!targetEmail in req.query) {
    res.render('error', { "errorMessage": "The target email is undefined." });
}
else {
  ...
}
```
As suggested in the comment you pointed out, note 2:
```
Some people consider this behavior confusing, arguing that it leads to hard-to-find errors and recommend using in operator instead
```
To me, using `in` "reads" really well. 

Thanks for pointing this out, really interesting!

Comment (sergii-tkachenko) [inline mbc-user-subscriptions-server.js]: Yes, this is perfectly fine for me.

**Comment by sergii-tkachenko** [comment]
:+1:

**Pull request merged**